mlir/tools/lmns-mlir-opt/lmns-mlir-opt.cpp

```diff
@@ -0,0 +1,203 @@
+//===- mlir-opt.cpp - MLIR Optimizer Driver -------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// Main entry function for mlir-opt for when built as standalone binary.
+//
+//===----------------------------------------------------------------------===//
```

Comment:
This has not been resolved.

Reply:
this is resolved, Idl why its not showing here:
`// This is a wrapper around mlir-opt. It is used to extract luminous module
// nested inside a module. It will dump out the host code to stderr and the
// device code to stdout. The command line options will be forwarded to mlir-opt
// along with the device code.
//
// TODO: This will be removed in the future and incorporated as a pass in mlir-opt
// something like --luminous-extract-module="outfile=<file>"`

Reply:
Ok I do see it now.  Although the title is wrong at the top